cx/slack/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "slack-cx-integration",
+  "version": "0.0.1",
+  "description": "A server that integrates Dialogflow CX with Slack.",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "test": "mocha --recursive --exit"
+  },
+  "dependencies": {
+    "@google-cloud/dialogflow-cx": "2.7.0",
+    "@slack/events-api": "^2.3.4",
+    "@slack/web-api": "^5.15.0",
+    "body-parser": "^1.19.0",
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.7"
```

Comment:
Just curious whether all the dependencies here are required, because some of them are not included/used in `server.js`. If not, probably remove the unnecessary dependencies.

Reply:
I left detenv and nodemon for the development side. body-parser is not currently being used but I though it might be needed when working with rich responses, if that's not the case I can remove it.